fem/libceed/solvers-utility.h

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2010-2020, Lawrence Livermore National Security, LLC. Produced
```

Comment:
Can you rename this file to `util.hpp`, I don't think these functions are specific to solvers.